src/Tasks/LDAPMemberSyncOneTask.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace SilverStripe\LDAP\Tasks;
+
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\LDAP\Services\LDAPService;
+use SilverStripe\Security\Permission;
+
+/**
+ * Class LDAPMemberSyncOneTask
+ * @package SilverStripe\LDAP\Tasks
+ *
+ * Debug build task that can be used to sync a single member by providing their email address registered in LDAP.
+ *
+ * Usage: /dev/tasks/LDAPMemberSyncOneTask?mail=john.smith@example.com
+ */
+class LDAPMemberSyncOneTask extends LDAPMemberSyncTask
+{
+    /**
+     * {@inheritDoc}
+     * @var string
+     */
+    private static $segment = 'LDAPMemberSyncOneTask';
+
+    /**
+     * @var array
+     */
+    private static $dependencies = [
+        'ldapService' => '%$' . LDAPService::class,
+    ];
+
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return _t(__CLASS__ . '.SYNCONETITLE', 'Sync single user from LDAP');
+    }
+
+    /**
+     * Syncs a single user based on the email address passed in the URL
+     *
+     * @param HTTPRequest $request
+     */
+    public function run($request)
+    {
+        $email = $request->getVar('email');
+
+        if (!$email) {
+            die('You must supply an email parameter to this method.');
+        }
+
+        $user = $this->ldapService->getUserByEmail($email);
+
+        if (!$user) {
+            die(sprintf('No user found in LDAP for email %s', $email));
```

Comment:
Maybe we could use `$this->log()` and then return instead of die? It's super minor but I know there won't be an EOL in this

Reply:
Yeah and if you _really_ want to stop PHP's execution, use `exit` instead, simply because from a dev's perspective, `die` looks like debug code that's been left-in!

Reply:
I've updated it to use `exit` since there's an existing example of the same kind of direct output above. It's OK for now!